RocketChat/Rocket.Chat
# [FIX] Don't ask again modals blinking

## Proposed changes (including videos or screenshots)


Made the check before opening the modal.


## Issue(s)


## Steps to test or reproduce

On the sidebar, hover a room, click the menu and then choose "Hide"

A modal should open, mark the checkbox "Do not ask again"

Try to hide another room. The confirmation modal will popup and close instantly, confirming the action.
## Further comments
